lib/routers/establishment/projects.js

```diff
@@ -85,41 +91,41 @@ router.param('id', (req, res, next, id) => {
     .then(() => Project.query().findById(id).where('establishmentId', req.establishment.id))
     .then(project => {
       return ProjectVersion.query()
```

Comment:
We _probably_ want an `if (!project) throw new NotFoundError()` here. Unless that gets caught further up the stack?

Reply:
right you are